hypha/apply/funds/models/forms.py

```diff
@@ -211,9 +211,46 @@ def __str__(self):
         return self.form.name
 
 
+class AbstractRelatedProjectSOWForm(Orderable):
+    class Meta(Orderable.Meta):
+        abstract = True
+
+    form = models.ForeignKey('application_projects.ProjectSOWForm', on_delete=models.PROTECT)
+
+    @property
+    def fields(self):
```

Comment:
While abstracting a foreign key as a class property, make it feel like an non-expensive access, while in reality it's not. 

I would suggest converting it to `@cached_property` or just not convert it to a property at all.

Reply:
It is the same for other forms, I just copied it from there to keep things similar. Can we work on it in different PR as an improvement for all the forms?